plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1178,6 +1184,9 @@ public void setTableProperties(ConnectorSession session, ConnectorTableHandle ta
 
         try {
             updateProperties.commit();
```

Comment:
In case of partitioning property we do a commit without doing any update. This sounds not right to me.

Reply:
Fixed. Make the UpdateProperties optional